.github/workflows/check-excluded-packages.yml

```diff
@@ -1,7 +1,14 @@
 name: Check excluded packages
 
 on:
-  pull_request
+  pull_request:
```

Comment:
I was wondering if we could start triggering this (and other) auxiliary with `path` argument - i.e. on changes in the listed directories? As it stands it will always be triggered

Reply:
Hm, I think I wouldn't go for that. It seems much worse to not build by accident (because the `path` is incorrect) than to build when not needed. I suspect most changes are code-related after all. If we notice that there is a lot of builds that only touch some non-code files we could exclude those files specifically in the future.